Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:alpine
+RUN apk add --update gcc libc-dev linux-headers
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY src/ src/
```

Comment:
Makes perfect sense! Thanks.

Reply:
Only issue with this is that I don't know how to copy just requirements and the src directory with a single copy...

I am worried that 
```
COPY requirements.txt src/ . 
```
will put the contents of src into the root...

Any suggestions? Otherwise I would leave it as it is.

Reply:
So I managed to do it with a dockerignore file and `COPY . .` as described above!